### ISSUE TITLE
Even if the init using java_selenium_maven is failing getting a successful message from gauge

### ISSUE THREAD

**Issue by sguptatw** [getgauge/gauge#391]
**Expected Behavior**
The message from gauge should be according to the exit code of the template being executed.

**Actual Behavior**
Even if the init using java_selenium_maven is failing getting a successful message from gauge

**Steps to replicate**
1. Download the attached project
[CounterWebApp.zip](https://github.com/getgauge/gauge/files/258658/CounterWebApp.zip)

2. cd CounterWebApp
3. now run `gauge --init java_selenium_maven`. give dummy values to proceed.
It fails as it is not able to add a child maven project.
4. However we get a successful message from gauge

**Version**
```
Gauge version: 0.4.1.nightly-2016-05-09

Plugins
-------
html-report (2.1.1.nightly-2016-05-10)
java (0.4.0.nightly-2016-05-09)
ruby (0.2.0.nightly-2016-05-09)
xml-report (0.1.2.nightly-2016-05-09)
```

**Comment by apoorvam** [comment]
Fix should be available in nightly >= 03.06.2016.

**Comment by sguptatw** [comment]
In the success message
```
Successfully initialized the project. Run specifications with "gauge specs/"
```
do mention that the command should be executed from the project dir.
